test/e2e/run_volume_test.go

```diff
@@ -625,7 +625,7 @@ VOLUME /test/`, ALPINE)
 
 		session = podmanTest.Podman([]string{"run", "--rm", "-v", ".:/app:O", ALPINE, "ls", "/app"})
 		session.WaitWithDefaultTimeout()
-		Expect(session.OutputToString()).To(ContainSubstring(filepath.Base(CurrentSpecReport().FileName())))
+		Expect(session.OutputToString()).To(ContainSubstring(" quadlet "))
```

Comment:
rpm package includes a compiled ginkgo binary, but no source files. I could `or(source or binary)`, but this seems good enough.